Library/Homebrew/formula.rb

```diff
@@ -405,11 +418,30 @@ def linked_keg
     Pathname.new("#{HOMEBREW_LIBRARY}/LinkedKegs/#{name}")
   end
 
+  def latest_head_prefix
+    head_prefixes = installed_prefixes.select do |p|
+      p.basename.to_s.start_with?("HEAD")
+    end
+
+    head_prefixes.sort_by do |p|
+      Tab.for_keg(Keg.new(p)).source_modified_time
+    end.last
+  end
+
+  # Check if stable was updated since last the prefix was installed.
+  def prefix_up_to_date?(prefix)
```

Comment:
I think this should be named as `head_prefix_up_to_date?`, because it is only used by head.